frontend/integration-tests/protractor.conf.ts

```diff
@@ -187,6 +192,7 @@ export const config: Config = {
       'tests/deploy-image.scenario.ts',
       'tests/performance.scenario.ts',
       'tests/monitoring.scenario.ts',
+      'tests/storage.scenario.ts',
```

Comment:
I would move this up below secrets scenario

Reply:
Thanks for your suggestion, will update soon.